resolwe_bio/tools/make_igv_session_archive.py

```diff
@@ -1,38 +1,60 @@
 #!/usr/bin/env python3
-# pylint: disable=missing-docstring,invalid-name,import-error
 """Create IGV session for archive."""
 
 import argparse
-from lxml import etree
 import os
 
-parser = argparse.ArgumentParser(description='Create igv session')
+from lxml import etree
+
+parser = argparse.ArgumentParser(description="Create igv session.")
 
-parser.add_argument("-f", "--files", nargs='+', required=True, help="List of files")
-parser.add_argument("-g", "--genome", type=str, required=True, help="Genome")
+parser.add_argument('-f', '--input_file', required=True, help="File with paths to files for IGV")
 
 args = parser.parse_args()
 
-Global = etree.Element(
-    'Global',
-    genome=args.genome,
-    version='8'
-)
 
-Resources = etree.SubElement(Global, 'Resources')
+def make_xml_tree(input_file):
+    """Make xml tree for IGV session."""
+    global_ = etree.Element(  # pylint: disable=no-member
+        'Global',
+        version='3',
+    )
+
+    resources = etree.SubElement(global_, 'Resources')  # pylint: disable=no-member
+
+    with open(input_file, 'r') as tfile:
+        for line in tfile:
+            element_name = os.path.basename(line).rstrip()
```

Comment:
Since you use `element_name` and `element_path` only once, you can define values where they are needed and avoid cretaing these variables. However, you can "save" one `rstrip` command with `line = line.rstrip()`. Something like this::

    for line in tfile:
        line = line.rstrip()
        etree.SubElement(  # pylint: disable=no-member
            resources,
            'Resource',
            name=os.path.basename(line),
            path=os.path.join('..', line),
        )